src/main/resources/application.properties

```diff
@@ -4,5 +4,9 @@ spring.datasource.url=jdbc:mysql://localhost:3306/crm
 spring.datasource.username=root
 spring.datasource.password=moyagjhit911
```

Comment:
I believe there is a way to hide passwords and don't store them at github :)
Not in this branch, later... Create a separate "jira" for this. In description, add links
https://www.ricston.com/blog/encrypting-properties-in-spring-boot-with-jasypt-spring-boot/
http://stackoverflow.com/questions/37404703/spring-boot-how-to-hide-passwords-in-properties-file
At the end, if we have time, we should encrypt passwords somehow.